apps/eDEX-UI/install-64

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+rm -rf ~/eDEX-UI-RPi &>/dev/null
+mkdir ~/eDEX-UI-RPi
+wget https://github.com/GitSquared/edex-ui/releases/download/v2.2.6/eDEX-UI-Linux-arm64.AppImage -O ~/eDEX-UI-RPi/eDEX-UI-Linux-arm64_2.2.6.AppImage || error 'Failed to download 2.2.6 appimage!'
+chmod +x ./'eDEX-UI-RPi/eDEX-UI-RPi/eDEX-UI-Linux-arm64_2.2.6.AppImage' || error 'Failed to mark the appimage as executable!'
+
+echo "[Desktop Entry]
+Name=eDEX-UI 2.2.6
+Comment=eDEX-UI sci-fi interface
+Exec=bash -c "\""sudo modprobe fuse; $HOME/eDEX-UI-RPi/eDEX-UI-Linux-armv7l_2.2.6.AppImage"\"" %U
+Terminal=false
+Type=Application
+Icon=$(dirname "$0")/icon-64.png
+StartupWMClass=eDEX-UI
+X-AppImage-Version=2.2.6
+Categories=System;
+X-Desktop-File-Install-Version=0.23
+X-AppImage-Comment=Generated by /tmp/.mount_eDEX-UDQYwse/AppRun" > ~/.local/share/applications/appimagekit-edex-ui.desktop
```

Comment:
Just to note, AppImages has usually a Desktop file inside of them, which you should rather use that the one generated by this script.

I believe AppImage runtime supports exporting files from the AppImages like this:
```sh
<path-to-appimage> --appimage-export [file]
```

Reply:
so I have to find the name of the desktop file inside the appimage (if present) and then run something like this: `edex-ui-2.2.6.appimage --appimage-export name-of-desktop-shortcut.desktop`?

Reply:
@SpacingBat3 this didn't work, its says that it isn't suppported yet on that appimage.

Reply:
> @SpacingBat3 this didn't work, its says that it isn't suppported yet on that appimage.

Oh, I've also checked this and it seems there's an `--appimage-extract` flag, but no `--appimage-export`.

The syntax should be the same through.

Reply:
I won't use this because I'll have to edit the .desktop file anyway.